docs/descriptors/xb1sx.md

```diff
@@ -0,0 +1,167 @@
+0xC0,              //   End Collection
+0x05, 0x01,        //   Usage Page (Generic Desktop Ctrls)
+0x09, 0x32,        //   Usage (Z)
+0x15, 0x00,        //   Logical Minimum (0)
+0x26, 0xFF, 0x03,  //   Logical Maximum (1023)
+0x95, 0x01,        //   Report Count (1)
+0x75, 0x0A,        //   Report Size (10)
+0x81, 0x02,        //   Input (Data,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x00,        //   Logical Maximum (0)
+0x75, 0x06,        //   Report Size (6)
+0x95, 0x01,        //   Report Count (1)
+0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x05, 0x01,        //   Usage Page (Generic Desktop Ctrls)
+0x09, 0x35,        //   Usage (Rz)
+0x15, 0x00,        //   Logical Minimum (0)
+0x26, 0xFF, 0x03,  //   Logical Maximum (1023)
+0x95, 0x01,        //   Report Count (1)
+0x75, 0x0A,        //   Report Size (10)
+0x81, 0x02,        //   Input (Data,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x00,        //   Logical Maximum (0)
+0x75, 0x06,        //   Report Size (6)
+0x95, 0x01,        //   Report Count (1)
+0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x05, 0x01,        //   Usage Page (Generic Desktop Ctrls)
+0x09, 0x39,        //   Usage (Hat switch)
+0x15, 0x01,        //   Logical Minimum (1)
+0x25, 0x08,        //   Logical Maximum (8)
+0x35, 0x00,        //   Physical Minimum (0)
+0x46, 0x3B, 0x01,  //   Physical Maximum (315)
+0x66, 0x14, 0x00,  //   Unit (System: English Rotation, Length: Centimeter)
+0x75, 0x04,        //   Report Size (4)
+0x95, 0x01,        //   Report Count (1)
+0x81, 0x42,        //   Input (Data,Var,Abs,No Wrap,Linear,Preferred State,Null State)
+0x75, 0x04,        //   Report Size (4)
+0x95, 0x01,        //   Report Count (1)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x00,        //   Logical Maximum (0)
+0x35, 0x00,        //   Physical Minimum (0)
+0x45, 0x00,        //   Physical Maximum (0)
+0x65, 0x00,        //   Unit (None)
+0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x05, 0x09,        //   Usage Page (Button)
+0x19, 0x01,        //   Usage Minimum (0x01)
+0x29, 0x0B,        //   Usage Maximum (0x0B)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x01,        //   Logical Maximum (1)
+0x75, 0x01,        //   Report Size (1)
+0x95, 0x0B,        //   Report Count (11)
+0x81, 0x02,        //   Input (Data,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x00,        //   Logical Maximum (0)
+0x75, 0x01,        //   Report Size (1)
+0x95, 0x05,        //   Report Count (5)
+0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x05, 0x0C,        //   Usage Page (Consumer)
```

Comment:
This is probably the logo button. Thus, your descriptor won't report it in the 11 button bits above but in a separate report. That doesn't hurt, tho. We can still pretend 12 buttons above and live with a single blind bit. `xpadneo_usage_maps` will take care of this then and report the button. This usage page will probably report as `0x10085` or `0xC0223`. If not, add it to the struct. If it would be reported twice, add an ignore to the struct.